railties/test/generators/generated_attribute_test.rb

```diff
@@ -57,9 +58,11 @@ def test_field_type_returns_file_field
     end
   end
 
-  def test_field_type_with_unknown_type_returns_text_field
+  def test_field_type_with_unknown_type_raises_error
     %w(foo bar baz).each do |attribute_type|
-      assert_field_type attribute_type, :text_field
+      assert_raise Thor::Error do
```

Comment:
Is `Thor::Error` the right exception here? I wonder if there's something more specific we can grab 🤔

Reply:
Good point!
I've changed it to Rails::Generators::Error (which inherits from Thor::Error).
This is also used by other generators.

Reply:
I was wondering what SchemaDumper raises and it's just `StandardError` 😮 

https://github.com/rails/rails/blob/d29f96233d7b467a5866915e3d3fb3c7598ccb63/activerecord/lib/active_record/schema_dumper.rb#L155